examples/api/test/cupertino/dialog/cupertino_alert_dialog.0_test.dart

```diff
@@ -22,4 +23,49 @@ void main() {
     await tester.pumpAndSettle();
     expect(find.text(actionText), findsNothing);
   });
+
+  testWidgets('Check for Directionality', (WidgetTester tester) async {
+    const String actionText = 'Yes';
+    await tester.pumpWidget(const example.AlertDialogApp());
+
+    // Launch the CupertinoAlertDialog.
+    await tester.tap(find.byType(CupertinoButton));
+    await tester.pump();
+    await tester.pumpAndSettle();
+    expect(find.text(actionText), findsOneWidget);
+
+    // Tap on an action to close the CupertinoAlertDialog.
+    await tester.tap(find.text(actionText));
+    await tester.pumpAndSettle();
+    expect(find.text(actionText), findsNothing);
+  });
+
+  testWidgets('Check for Directionality', (WidgetTester tester) async {
+    Future<void> pumpWidget({required bool isLTR}) async {
+      await tester.pumpWidget(
+        CupertinoApp(
+          home: Directionality(
+            textDirection: isLTR ? TextDirection.ltr : TextDirection.rtl,
+            child: const CupertinoAlertDialog(
+              actions: <CupertinoDialogAction>[
+                CupertinoDialogAction(isDefaultAction: true, child: Text('No')),
+                CupertinoDialogAction(child: Text('Yes')),
+              ],
+            ),
+          ),
+        ),
+      );
+    }
+
+    await pumpWidget(isLTR: true);
+    Vector3 yesButton =
+        tester.firstRenderObject(find.text('Yes')).getTransformTo(null).getTranslation();
```

Comment:
Are you simply trying to query the position of the text? If so why can't you simply use 
```dart
    tester.getCenter(find.text('Yes'))
```